lib/rules/no-implicit-coercion.js

```diff
@@ -278,8 +301,10 @@ module.exports = {
                 operatorAllowed = options.allow.includes("!!");
                 if (!operatorAllowed && options.boolean && isDoubleLogicalNegating(node)) {
                     const recommendation = `Boolean(${sourceCode.getText(node.argument.argument)})`;
+                    const variable = astUtils.getVariableByName(sourceCode.getScope(node), "Boolean");
+                    const booleanExists = Boolean(variable?.identifiers.length === 0);
```

Comment:
```suggestion
                    const booleanExists = variable?.identifiers.length === 0;
```

`Boolean()` doesn't seem necessary here since `===` always evaluates to a boolean value?

Reply:
Oh, it should not have the `===`, yes